### PR TITLE
rsdk changes for widgets used in wavi

### PR DESCRIPTION
**Pull request by murali-shris**
**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: at_backupkey_flutter/lib/services/backupkey_service.dart [hunk 1]
```diff
@@ -9,17 +9,15 @@ class BackUpKeyService {
...
-  Future<Map<String, String>> getEncryptedKeys(String atsign) async {
+  static Future<Map<String, String>> getEncryptedKeys(String atsign) async {
     var result;
     try {
-      result = await atClientService.getEncryptedKeys(atsign);
-      result[atsign] = await this._getAESKey(atsign);
+      result = await KeyChainUtil.getEncryptedKeys(atsign);
+      result[atsign] = await _getAESKey(atsign);
     } catch (e) {
       result = {};
```
Comment (VJag) [inline at_backupkey_flutter/lib/services/backupkey_service.dart:22]: Let's add a log here. We are swallowing an exception.

File: at_follows_flutter/lib/services/connections_service.dart [hunk 2]
```diff
@@ -429,15 +430,14 @@ class ConnectionsService {
...
 
-  Future<bool> startMonitor() async {
-    var result = await _sdkService.startMonitor(acceptStream);
-    if (result) {
-      isMonitorStarted = true;
-      return true;
-    }
-    return false;
+  bool startMonitor() {
+    AtClientManager.getInstance().notificationService.subscribe().listen((notification) {
```
Comment (VJag) [inline at_follows_flutter/lib/services/connections_service.dart:434]: This returns notification service for current @sign, right?

Comment (VJag) [reply at_follows_flutter/lib/services/connections_service.dart:434]: This "ConnectionsService" should be deprecated

File: at_follows_flutter/lib/services/sdk_service.dart [hunk 3]
```diff
@@ -7,7 +7,6 @@ import 'package:at_follows_flutter/utils/app_constants.dart';
 import 'package:at_follows_flutter/utils/strings.dart';
 import 'package:at_server_status/at_server_status.dart';
 import 'package:at_utils/at_logger.dart';
-
 class SDKService {
```
Comment (VJag) [inline at_follows_flutter/lib/services/sdk_service.dart:10]: This should be deprecated

File: at_follows_flutter/lib/services/sdk_service.dart [hunk 4]
```diff
@@ -26,30 +25,30 @@ class SDKService {
...
-    return await _atClientServiceInstance.getPkamPrivateKey(atsign).timeout(
+    return await KeyChainUtil.getPkamPrivateKey(atsign).timeout(
         Duration(seconds: AppConstants.responseTimeLimit),
         onTimeout: () => _onTimeOut());
   }
 
   ///Returns `true` on deleting [atKey].
   Future<bool> delete(AtKey atKey) async {
-    return await _atClientServiceInstance.atClient!.delete(atKey).timeout(
+    return await AtClientManager.getInstance().atClient.delete(atKey).timeout(
```
Comment (VJag) [inline at_follows_flutter/lib/services/sdk_service.dart:44]: Do we need that null safety change? "!"

**Review by sitaram-kalluri — APPROVED** [review]

**Pull request merged**